### DISCUSSION TITLE
Bruno CLI output format

### DISCUSSION THREAD

**Discussion by eddimcbenz** [usebruno/bruno#1068] [Q&A] [answered]
I just switched from Postman to Bruno, because Bruno looks very promissing. By the way, you all are doing a great job!

But now I came to a point where I have no adequate solution with Bruno. I use Bruno CLI with Jenkins like I used Postman newman with Jenkins in the passed. This works till now really good. But the output format of the json file that is produced by Bruno CLI is not working with Jenkins in the way I used to have it integrated with Postmans newman CLI. With newman, there was the possibility to have a second reporter that created junit files.

Are different reporters maybe planned to be integrated into Bruno CLI, or does somebody have an idea how to get the json format of the output file transformed to a junit format?

Thank you in advance for help and an answer.
Eddi

**Comment by awinder** [comment, marked as answer]
Make sure you upgrade to the latest version of bruno (npm update -g @usebruno/cli if you've already installed, npm install -g @usebruno/cli if a new install). You should be able to use the --format flag in the cli:

bru run --output results.xml --format junit
bru run --help # to see more options

confirm that bru --version is > 1.3.0 if you're still not seeing the junit format option

**Reply by eddimcbenz** [reply]
Yes, thank you @awinder. I just updated to a newer version because I saw this new feature. I was testing it and it looked fine. Thank for the hint and for the implementation.

**Reply by ivycristina** [reply]
hello, I am using bruno-cli and wanted to generate a reports of the tests in html does it have this option?

**Reply by awinder** [reply]
Not directly in this tool but I've used the excellent https://github.com/inorton/junit2html tool (including when I was working on adding this functionality to help validate the Junit output). There's also a Go tool @ https://pkg.go.dev/github.com/alexec/junit2html#section-readme that looks promising, might be worth trying both out. So using one of those 2 tools, you'd want to run the Junit report in bruno to generate a Junit xml file, and then run the html generator tool (whichever junit2html version you choose) on it.

**Comment by sushmithasv** [comment]
Can we integrate report portal agent with usebruno cli?

**Comment by sushmithasv** [comment]
Can we pass multiple reporters in the command like below 
bru run --env <env file> --output results.json --output resulsts.xml --format junit

**Reply by helloanoop** [reply]
Tracking it at https://github.com/usebruno/bruno/issues/2918
